hubspot/metadata.go

```diff
@@ -10,19 +10,29 @@ import (
 	"github.com/spyzhov/ajson"
 )
 
+type ObjectMetadataResult struct {
```

Comment:
can you lower case this and `ObjectMetaDataError` since they are not used outside of this package?

Reply:
Fixed